packages/integration-karma/test/light-dom/basic/index.spec.js

```diff
@@ -0,0 +1,24 @@
+import { createElement, setFeatureFlagForTest } from 'lwc';
```

Comment:
Other tests worth adding to the suite:
- A test checking the `LightningElement.shadow` property is set to `true`.
- A test checking that `this.template` from within the component is `null` with light DOM.
- A test with multiple levels of inheritance where `Child` extends from `Base` which extends from `LightningElement` where `Base` sets shadow to true and `Child` sets it to false.
- Because of the class field loose mode I would also add a test where  `shadow` is defined as a class field getter and not a class field property: `get shadow() { return false }`.

As part of this PR, I also would like to see engine-server tests.